src/patternfly/components/Card/card.scss

```diff
@@ -110,6 +114,10 @@
   background-color: var(--pf-c-card--BackgroundColor);
   box-shadow: var(--pf-c-card--BoxShadow);
 
+  &.pf-m-description-list-group {
```

Comment:
Hmm... this is fine, but I wonder if this should be either:

1. If this is specific to the description list, move these styles into the description list. The `--pf-c-card...Padding` vars should work fine since you have `.pf-c-card` on the element the modifier applies to.
  * Going forward, if the DL card styles grow, it might seem odd to bundle a bunch of extra description list specific styles when someone is just using the card.
1. A generic thing for a card without children, with padding. Maybe `.pf-m-padding`? Seems redundant since that's what `<div class="pf-c-card"><div class="pf-c-card__body">...</div></div>` does now.
1. Styled in the DL as `.pf-c-description-list > .pf-c-card`. That seems inline with how we style other things like that - like the toolbar in the masthead. Given the markup constraints to get this to work with a DL, seems like selector should be fine?